rust/common-wit/src/lib.rs

```diff
@@ -7,36 +7,33 @@ use std::{collections::BTreeMap, str::FromStr};
 
 use serde::{Deserialize, Serialize};
 
-/// WIT definition for `common:module`
-pub const COMMON_MODULE_WIT: &[u8] =
-    include_bytes!("../../../typescript/common/module/wit/module.wit");
+/// WIT definition for `common:function`
+pub const COMMON_FUNCTION_WIT: &[u8] =
+    include_bytes!("../../../typescript/common/function/wit/function.wit");
 
 /// WIT definition for `common:io`
 pub const COMMON_IO_WIT: &[u8] = include_bytes!("../../../typescript/common/io/wit/io.wit");
 
 /// WIT definition for `common:data`
 pub const COMMON_DATA_WIT: &[u8] = include_bytes!("../../../typescript/common/data/wit/data.wit");
 
-/// WIT definition for `common:script`
-pub const COMMON_SCRIPT_WIT: &[u8] =
-    include_bytes!("../../../typescript/common/script/wit/script.wit");
-
 /// A target that some candidate source code may express the implementation of
 #[derive(Clone, Copy, Debug, Serialize, Deserialize)]
 pub enum Target {
     /// The most basic target: a Common Module
-    #[serde(rename = "common:module")]
-    CommonModule,
+    #[serde(rename = "common:function/module")]
+    CommonFunction,
     /// Effectively the same as a Common Module, but intepreted w/o a compile step
```

Comment:
A few "Common Module" in comments, may need some updates

Reply:
In this case it is intended. Module is an umbrella.